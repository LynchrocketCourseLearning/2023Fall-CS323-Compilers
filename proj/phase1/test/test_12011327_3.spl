int test_12011327_3()
{
	int 12nihao = 12;
  return @;
}
