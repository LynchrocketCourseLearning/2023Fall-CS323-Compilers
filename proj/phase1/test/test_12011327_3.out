Error type A at Line 3: Identifier starts with digits 12nihao
Error type A at Line 4: Mysterious lexeme @
